UnitTests/CatalogTests/ServiceTests/CommentsServiceTest/AddCommentTest.cs

```diff
@@ -0,0 +1,77 @@
+﻿using Catalog.Services.Comments;
+using Common.Interfaces;
+using Common.Models.Comments;
+using FluentValidation;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Moq;
+using System.Threading.Tasks;
+
+namespace UnitTests.CatalogTests.ServiceTests.CommentsServiceTest
+{
+
+    [TestClass]
+    public class AddCommentTests
+    {
+        private Mock<IDataAccessor<Comment>> CommentsAccessorMock;
```

Comment:
This is rather Stub than Mock. I've also made this mistake with the naming before, so I propose you to read the article about the differences between Stubs and Mock. The difference is quite subtle and elusive but can prevent us later on from the proper testing process.
Unfortunately, I don't have a link to the source to read from right now, but there's a good article on an MSDN covering the topic.
In case you will need my assistance in it, please go ahead and contact me and we will discuss this together.

- [ ] Learn more about the Stubs and Mocks.
- [ ] Make adjustments to the variable names accordingly.